canonical-schema.json

```diff
@@ -86,6 +86,7 @@
                 , "property"    : { "$ref": "#/definitions/property"    }
                 , "input"       : { "$ref": "#/definitions/input"       }
                 , "expected"    : { "$ref": "#/definitions/expected"    }
+                , "timeout"     : { "$ref": "#/definitions/timeout"     }
```

Comment:
The definition that is referenced by `#/definitions/timeout` would need to be added also.

Reply:
got it.